packages/website/src/docs/divider.mdx

```diff
@@ -1,16 +1,27 @@
 ---
 title: Divider
 component: Divider
+description: Useful for displaying a visual separator between parts of content.
 ---
 
-```js live=true
+```js live
 <Divider />
 ```
 
-```js live=true
-<Divider variant="vertical" height={50} />
+### variant
+
+<Prop isOptional default="0">
+  0 | 1 | 2
+</Prop>
+
+There are 3 variants to choose from
+
+```js live
+<Divider size={0} />
```

Comment:
I'm confused with this one, it says 0 is the default size, but looking at the example, it looks like 0 produces a really small one. However above, when not passing a `size` at all, it's full width.

Reply:
Actually there's no default for this. Maybe the API needs improving, but I updated the docs to reflect the current API.

Reply:
Yeah maybe remove the default, keep optional but also precise that if no value is passed in, it will be full width.